packages/opentelemetry-api/package.json

```diff
@@ -63,7 +63,7 @@
     "nyc": "^14.1.1",
     "ts-loader": "^6.0.4",
     "ts-mocha": "^6.0.0",
-    "tslint-consistent-codestyle": "^1.15.1",
+    "tslint-consistent-codestyle": "^1.16.0",
```

Comment:
We should update all the packages with the latest version, I will open follow-up PR to do that.

Reply:
Looks like you already did that here, please ignore the comment.